cumulus/pallets/executive/src/lib.rs

```diff
@@ -120,11 +120,12 @@ use codec::{Codec, Encode};
 use frame_support::{
 	dispatch::PostDispatchInfo,
 	traits::{
-		EnsureInherentsAreFirst, ExecuteBlock, OffchainWorker, OnFinalize, OnIdle, OnInitialize,
-		OnRuntimeUpgrade,
+		EnsureInherentsAreFirst, ExecuteBlock, Get, OffchainWorker, OnFinalize, OnIdle,
+		OnInitialize, OnRuntimeUpgrade,
 	},
 	weights::{DispatchClass, DispatchInfo, GetDispatchInfo},
 };
+pub use pallet::*;
```

Comment:
Please import specific items instead, it helps with readability

Reply:
It's the substrate idiom, but I can change it for sure if you insist.